src/components/opening-hours/opening-hours.vue

```diff
@@ -68,6 +76,49 @@ export default {
   },
   computed: {
     ...mapState(['isMobile']),
+    timeSlots() {
+      const { openingHoursSpace, openingHoursBuilding } = this
+      const newOpeningHours = openingHoursSpace.map((openingHour, index) => {
+        const buildingStart = openingHoursBuilding[index].time.length ? openingHoursBuilding[index].time[0][0] : []
+        const buildingEnd = openingHoursBuilding[index].time.length ? openingHoursBuilding[index].time[0][1] : []
+        const newTimeslots = []
+
+        newTimeslots.day = openingHour.day
+
+        newTimeslots.times = openingHour.time.reduce((timeslots, timeslot, index) => {
+          // Check if there is a busy slot before the first open time slot
+          if (index === 0 && new Date(timeslot[0]) > new Date(buildingStart)) {
+            const start = buildingStart
+            const end = timeslot[0]
+            timeslots.push({ time: [start, end], busy: false })
```

Comment:
`busy` should be `true` here right? When a space is available later than the building opens, it means it's occupied.